samples/net/download/prj.conf

```diff
@@ -4,18 +4,18 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-CONFIG_NRF_MODEM_LIB=y
 
+CONFIG_DOWNLOAD_CLIENT=y
+CONFIG_DOWNLOAD_CLIENT_STACK_SIZE=4096
+
+# Networking
 CONFIG_NETWORKING=y
+# CONFIG_NET_SOCKETS_OFFLOAD=y
 CONFIG_NET_SOCKETS=y
-CONFIG_NET_NATIVE=n
-
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_TCP=y
+CONFIG_NET_NATIVE=y
```

Comment:
```
CONFIG_NET_TCP=y
CONFIG_NET_NATIVE=y
```
These two options should be removed from the `prj.conf` file and added to the project configuration for the WiFi boards. When using the LTE chips, we don't need the native (Zephyr's) network stack.